api/envoy/extensions/filters/network/connection_limit/v3/connection_limit.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.network.connection_limit.v3;
+
+import "envoy/config/core/v3/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.network.connection_limit.v3";
+option java_outer_classname = "ConnectionLimitProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Connection limit]
```

Comment:
This feature needs a release note. Please add it.

Reply:
Added it to the `docs/root/version_history/current.rst`.
Does any other notes I should add?